.github/workflows/ci.yml

```diff
@@ -0,0 +1,255 @@
+# Copyright 2020 Evan Miller
+# Copyright 2020 Matt Borland
```

Comment:
Shouldn't this file be part of a separate PR?

Reply:
Yes; this is identical to PR #476. I had it in here to speed up CI cycles but will pull it out.